msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/mwpj/TestCase2519783.kt

```diff
@@ -0,0 +1,115 @@
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.client.msal.automationapp.testpass.broker.mwpj
+
+import com.microsoft.identity.client.msal.automationapp.R
+import com.microsoft.identity.client.msal.automationapp.testpass.broker.AbstractMsalBrokerTest
+import com.microsoft.identity.client.ui.automation.annotations.LocalBrokerHostDebugUiTest
+import com.microsoft.identity.client.ui.automation.annotations.SupportedBrokers
+import com.microsoft.identity.client.ui.automation.broker.BrokerHost
+import com.microsoft.identity.client.ui.automation.rules.LoadLabUserTestRule
+import com.microsoft.identity.labapi.utilities.client.ILabAccount
+import com.microsoft.identity.labapi.utilities.client.LabQuery
+import com.microsoft.identity.labapi.utilities.constants.AzureEnvironment
+import com.microsoft.identity.labapi.utilities.constants.TempUserType
+import com.microsoft.identity.labapi.utilities.constants.UserType
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TestRule
+
+// https://identitydivision.visualstudio.com/Engineering/_workitems/edit/2519783
+@LocalBrokerHostDebugUiTest
+@SupportedBrokers(brokers = [BrokerHost::class])
+class TestCase2519783 : AbstractMsalBrokerTest() {
+
+    private lateinit var mUsGovLabAccount: ILabAccount
+
+    @get:Rule
+    val loadAdditionalLabUserRule: TestRule = LoadLabUserTestRule(getAdditionalLabQuery())
+
+    @Test
+    fun test_2519783() {
+
+        val username = mLabAccount.username
+        val password = mLabAccount.password
+        val usGovUsername = mUsGovLabAccount.username
+        val usGovPassword = mUsGovLabAccount.password
+        val brokerHostApp = broker as BrokerHost
+        brokerHostApp.enableMultipleWpj()
+        brokerHostApp.performDeviceRegistrationMultiple(username, password)
+        brokerHostApp.performDeviceRegistrationMultiple(usGovUsername, usGovPassword)
+        val deviceRegistrationRecords = (broker as BrokerHost).allRecords
+        brokerHostApp.installCertificateMultiple(deviceRegistrationRecords[0]["TenantId"] as String)
+        brokerHostApp.installCertificateMultiple(deviceRegistrationRecords[1]["TenantId"] as String)
+    }
+
+    override fun getLabQuery(): LabQuery {
+        return LabQuery.builder()
+                .userType(UserType.CLOUD)
+                .build()
+    }
+
+    override fun getTempUserType(): TempUserType? {
+        return null
+    }
+
+    private fun getAdditionalLabQuery(): LabQuery {
+        return LabQuery.builder()
+                .userType(UserType.CLOUD)
+                .azureEnvironment(AzureEnvironment.AZURE_US_GOVERNMENT)
+                .build()
+    }
+
+    @Before
+    fun before() {
+        mUsGovLabAccount = (loadAdditionalLabUserRule as LoadLabUserTestRule).labAccount
```

Comment:
If you just need to create a second user, there's a simpler way using mLabClient instead of creating a reference to the rule

Reply:
`// query to load a user from the same tenant that was used for WPJ
        final LabQuery query = LabQuery.builder()
                .userType(UserType.CLOUD)
                .build();

        // get username and password for this account
        final ILabAccount user2 = mLabClient.getLabAccount(query);
        final String username2 = user2.getUsername();
        final String password2 = user2.getPassword();`

Reply:
Using the same query returns the same account, what I can do is use the existing lab client to get different accounts providing the corresponding queries.

Reply:
https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1817/commits/359167a0d4f68b7d9a81027aa2481933be178e61

Reply:
Yes, you would use a different query to get a different account.